clients/ui/frontend/src/app/useSettings.tsx

```diff
@@ -21,24 +21,24 @@ export const useSettings = (): {
     let cancelled = false;
     const watchConfig = () => {
       Promise.all([fetchConfig(), fetchUser()])
-        .then(([config, user]) => {
+        .then(([fetchedConfig, fetchedUser]) => {
```

Comment:
@lucferbux I had to change the name of `config` and `user` here because the linter didn't like that we were reusing the same name for the state variables.

Reply:
well, that's why we are using linters, great catch hahaha